src/homepageExperience/components/steps/Tokens.tsx

```diff
@@ -62,11 +62,11 @@ export const Tokens: FC<OwnProps> = ({
   )
 
   useEffect(() => {
-    const fetchResources = async () => {
-      await dispatch(getAllResources())
+    const fetchResources = () => {
```

Comment:
This one is also interesting - and the issues with this `useEffect` hook have led us to a bug we need to fix in the wizards that generate tokens.

1. The `await` wasn't doing anything here, because there was no code following it in the hook.
2. Really, the bigger issue here is that dispatching the thunk middleware needs to have a `try/catch` block around it, or invoke the `.then()` and `.catch()` methods from the returned promise. `getAllResources` throws an error if it doesn't receive a `200` status code from the endpoint. Right now, that error is uncaught, so the UI will likely bug out on this page when the token request fails if we don't add error handling.

Reply:
@alexjw3 and I talked about this - this is a great one to talk about at code review.